### DISCUSSION TITLE
Multi-select using 'Select' or 'Combobox'

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#939] [Q&A] [answered]
Is there a multi-select use case using the Paste 'Select' or 'Combobox' component? 
The new Debugger concept has an export option that a customer can select multiple fields. (I try not to use the checkbox because of limited space. )
Thanks!

**Comment by kate-porter** [comment]
Hey @sang-yeo - sorry for the delay on this one!

Would you be able to share a link to the design treatment you are considering? Paste doesn't currently support multi-select in the Select or Combobox components, nor does the Combobox primitive support it. But if you could share the designs so we can get a better sense for what you are trying to accomplish, we can brainstorm how it might be built using Paste.

**Reply by sang-yeo** [reply]
Sure @katieporter, here you go. I thought about the checkbox option but it makes the modal really busy. 
<img width="824" alt="Screen Shot 2020-11-16 at 5 22 34 PM" src="https://user-images.githubusercontent.com/67388401/99328239-8338a180-2830-11eb-8eb9-498958304999.png">
A customer can select multiple fields.

**Comment by kate-porter** [comment, marked as answer]
Hey @sang-yeo - thanks for providing a screenshot, that's helpful.

So, the Combobox component unfortunately doesn't support multi-select yet. The Select component _technically_ does, though we generally don't recommend it, because the default browser interaction pattern is pretty strange. You can see an example of it in the last "do/don't" on the [Select docs page](https://paste.twilio.design/components/select#when-to-use-a-select) (it's in the "don't" column 😂).

For now, the best way to accomplish this using Paste will be to use the [Select All checkbox group](https://paste.twilio.design/components/checkbox#unchecked-checked-and-indeterminate). You can orient the checkboxes horizontally and/or put them into multiple columns, which may help the modal from feeling too busy.

Give that a try and let us know if that will work for you.
